### PR TITLE
Predictions visualizations

### PR DESCRIPTION
**Pull request by komalsingh0925**
In order to run predictions_test.py follow these steps:

1. Run the steps given here: https://github.com/trevorwinser/Infinite-Loopers/blob/main/Instructions/flask.md
2. FIx line 10 to make it to your path of the ChromeDriver Instance (or maybe just remove it if you have a Mac i'm not sure about Mac)
3. If you want to test the functionality of the overall graph setup go on http://127.0.0.1:5000/temperature-predictions
4. Note if the graph doesn't load for some months it means that there is not any or enough data for those months

#142

### CODE REVIEW

File: code/app.py [hunk 1]
```diff
@@ -4,15 +4,36 @@
...
 basedir = os.path.abspath(os.path.dirname(__file__))
 sqlite_setup.main()
-app = Flask(__name__)
 
+app = Flask(__name__)
 
+@app.route('/temperature-predictions', defaults={'month': None})
+@app.route('/temperature-predictions/<int:month>')
+def temperature_predictions(month):
+    conn = sqlite3.connect('hurriscan.db')
```
Comment (julieflament) [inline code/app.py:23]: add try/catch

File: code/app.py [hunk 2]
```diff
@@ -4,15 +4,36 @@
...
+    df = pd.read_sql_query("SELECT month, AVG(temp) AS avg_temp, AVG(humidity) AS avg_humidity, AVG(air) AS avg_air FROM Data GROUP BY month", conn)
```
Comment (julieflament) [inline code/app.py:24]: make a function that returns humidity, temperature, and air based on month

**Review by julieflament — CHANGES_REQUESTED** [review]
minor changes

File: code/app.py (hunk 1 above)
Comment (komalsingh0925) [reply code/app.py]: This is done now

File: code/app.py
Comment (trevorwinser) [inline code/app.py:1]: Traceback (most recent call last):
  File "C:\Users\trevo\OneDrive\Documents\Computer Science\COSC310\Infinite-Loopers\code\venv\Lib\site-packages\flask\cli.py", line 247, in locate_app
    __import__(module_name)
  File "C:\Users\trevo\OneDrive\Documents\Computer Science\COSC310\Infinite-Loopers\code\app.py", line 7, in <module>
    import pandas as pd
  File "C:\Users\trevo\OneDrive\Documents\Computer Science\COSC310\Infinite-Loopers\code\venv\Lib\site-packages\pandas\__init__.py", line 32, in <module>
    raise ImportError(
ImportError: Unable to import required dependencies:
numpy: Error importing numpy: you should not try to import numpy from
        its source directory; please exit the numpy source tree, and relaunch
        your python interpreter from there.
        
numpy import error. I will try to debug to see what's wrong.

**Review by julieflament — APPROVED** [review]
made necessary changes and tests run! good job!

File: code/app.py
Comment (trevorwinser) [reply code/app.py:1]: Fixed by uninstalling numpy and pandas

**Review by trevorwinser — APPROVED** [review]
Looks good!

**Pull request merged**